d2/src/main/java/com/linkedin/d2/xds/XdsToD2PropertiesAdaptor.java

```diff
@@ -273,6 +280,7 @@ public void onChanged(XdsClient.NodeUpdate update)
           }
           catch (PropertySerializationException e)
           {
+            _clusterEventBus.publishInitialize(clusterName, null); // still notify event bus to avoid timeout in case some subscribers are waiting for the data
```

Comment:
```suggestion

            // still notify event bus to avoid timeout in case some subscribers are waiting for the data
            _clusterEventBus.publishInitialize(clusterName, null);
```

Reply:
Done